src/Console/TerminateCommand.php

```diff
@@ -34,14 +34,13 @@ class TerminateCommand extends Command
     /**
      * Execute the console command.
      *
-     * @param  \Illuminate\Contracts\Cache\Factory  $cache
      * @param  \Laravel\Horizon\Contracts\MasterSupervisorRepository  $masters
      * @return void
      */
-    public function handle(CacheFactory $cache, MasterSupervisorRepository $masters)
```

Comment:
Why not simply reuse `$cache` instead of creating a breaking change to public method on a minor/patch release.

Reply:
i changed because this CacheFactory is just weird, none of the methods are there:

![image](https://github.com/laravel/horizon/assets/347400/830debcb-ea71-4665-a870-574d395e5b83)

The right one to use should be the Store contract... but anyway... i reverted this change